packages/ast-spec/tests/util/setupJest.ts

```diff
@@ -0,0 +1,8 @@
+import { addSerializer } from 'jest-specific-snapshot';
+import { serializers } from './serializers';
+
+for (const serializer of serializers) {
+  // the jest types are wrong here
```

Comment:
oh?

Reply:
I actually think this is fixed now

Reply:
This was marked as resolved a some time ago, but the comment still remains in the code so wanted to open it up again to a final resolution now